ChopChop/Views/OnlineRecipes/OnlineRecipeBySelfView.swift

```diff
@@ -1,7 +1,7 @@
 import SwiftUI
 
 struct OnlineRecipeBySelfView: View {
-    @ObservedObject var viewModel: OnlineRecipeBySelfViewModel
+    let viewModel: OnlineRecipeBySelfViewModel
```

Comment:
If the view model is no longer observed, does `OnlineRecipeViewModel` still need to be an observable object?

Reply:
In this particular view, OnlineRecipeBySelfViewModel does not need to be observed because it does not have any @Published properties.

OnlineRecipeViewModel still needs to be an observable object because another view, ie OnlineRecipeView, needs to observe OnlineRecipeViewModel